sha3/src/main/java/com/theromus/utils/HexUtils.java

```diff
@@ -93,5 +93,21 @@ public static String convertBytesToString(final byte[] data) {
 
         return new String(buffer.toByteArray());
     }
-
+    
+    /**
+     * 
```

Comment:
Please add short description.

Reply:
I forgot about this. 😅 
I saw that there was a mistake and it was unnecessarily complex, in the next commit will be fixed.

Reply:
Thanks!